files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -152,6 +152,13 @@
 #
 - command: systemctl mask systemd-journald-audit.socket
 
+#
+# The nfs-blkmap.service is disabled by default but since it is wanted
+# by nfs-client.target it will always get started.  We don't use pNFS
+# so mask the nfs-blkmap.service to keep it from running.
+#
+- command: systemctl mask nfs-blkmap.service
```

Comment:
Note that if we are running this on upgrade then it will not stop the service, i.e. it will keep running until we reboot the system or it is stopped via some dependencies. I don't know if that's an issue or not.

We can also stop it by running `systemctl mask --now nfs-blkmap.service`, however given that we also run this script in a chroot during the initial image creation, I'm not sure how it would react.

Reply:
if we want to do that, we'd likely have to prevent it from running during the build.. there's the "is chroot" flag that can be used to do that.

Reply:
Right, I guess we could do that as 2 commands:
```
- command: systemctl mask nfs-blkmap.service

- command: systemctl stop nfs-blkmap.service
  when: not ansible_is_chroot
```

Reply:
The service is in a perpetual "failed" state on Delphix Engines, so stopping it won't do anything, really. I think masking it so that the next time it tries to start, nothing will happen, seems like what we should do.